docs/getting-started/install_turtles_operator.md

```diff
@@ -4,4 +4,17 @@ sidebar_position: 4
 
 # Install Rancher Turtles Operator
 
-This section needs to be filled out.
\ No newline at end of file
+Rancher turtles requires to have a direct connection to the rancher manager cluster. This could be achieved by:
+1. Installing in in the same cluster as the rancher manager.
+
+Recommended path of installation for the operator is by using helm. To install it in your cluster you need to add repository first:
```

Comment:
What do you think to covering how to install it via the Rancher Manager UI as well?

Reply:
I think this is already covered in https://github.com/rancher-sandbox/rancher-turtles-docs/pull/19